File: apps/s-2-admin/src/api/workflowReminder/WorkflowReminder.ts
import { Booking } from "../booking/Booking";
import { WorkflowStep } from "../workflowStep/WorkflowStep";

export type WorkflowReminder = {
  id: number;
  booking?: Booking | null;
  method?: "EMAIL" | "SMS";
  scheduledDate: Date;
  referenceId: string | null;
  scheduled: boolean;
  workflowStep?: WorkflowStep;
};

File: apps/cactus-demo/src/APIs/WorkflowReminder/Dtos/WorkflowReminder.cs
using CactusDemo.Core.Enums;

namespace CactusDemo.APIs.Dtos;

public class WorkflowReminder
{
    public int Id { get; set; }

    public MethodEnum Method { get; set; }

    public DateTime ScheduledDate { get; set; }

    public string? ReferenceId { get; set; }

    public bool Scheduled { get; set; }

    public int? Booking { get; set; }

    public int WorkflowStep { get; set; }
}
